reporter-web-app/src/components/SummaryViewLicenses.js

```diff
@@ -79,9 +79,10 @@ const SummaryViewLicenses = (props) => {
                 sortOrder: sortedInfo.columnKey === 'name' && sortedInfo.order,
                 key: 'name',
                 render: (text, row) => (
-                    <Tag color={row.color}>
-                        {text}
-                    </Tag>
+                    <span>
+                        <Icon type="file-text" style={{ color: row.color }} />
+                        {` ${text}`}
```

Comment:
Why is this not just `{text}` as before? Is the leading space correct here? Shouldn't we rather use a right-side margin for the icon if you want to add some spacing between icon and text?